### ISSUE TITLE
Migrations configuration key "name" does not exist.

### ISSUE THREAD

**Issue by shepard8** [doctrine/migrations#1024]
### Bug Report

|    Q        |   A
|------------ | ------
| BC Break    | yes/no (I don't know what BC is, sorry :confused: )
| Version     | 3.0.1

#### Summary

Following the [documentation](https://www.doctrine-project.org/projects/doctrine-migrations/en/latest/reference/configuration.html#configuration), I used the following `migrations.php` file:

```php
<?php return [
  'name' => 'Test Migrations',
  'table_storage' => [
    'table_name' => 'test_migrations',
    'version_column_name' => 'version',
    'version_column_length' => '1024',
    'executed_at_column_name' => 'executed_at',
    'execution_time_column_name' => 'execution_time'
  ],
  'migrations_paths' => [
    'Test\Migrations' => '/home/fabian/migrationstest/Migrations'
  ],
  'all_or_nothing' => true,
  'check_database_platform' => true
]; ?>
```

However when running `./vendor/bin/doctrine-migrations generate` as indicated on the [next page](https://www.doctrine-project.org/projects/doctrine-migrations/en/latest/reference/migration-classes.html#migration-classes), I end up with the following error :
> Migrations configuration key "name" does not exist.

#### Current behavior

The error shown above

#### How to reproduce

Just `composer require 'doctrine/migrations'` in some empty folder, put the migrations.php file as above, and run the command above.

#### Expected behavior

New migration class should be generated

This is fixed by commenting out the `'name' => 'Test Migrations',` line. I guess this is just a documentation error and name is not relevant anymore, but I am completely new to doctrine migrations so I can't be sure at all.

**Comment by shepard8** [comment]
According to [this issue](https://github.com/doctrine/migrations/issues/1017), which is related to this issue but not the same (upgrading vs main documentation), this is indeed a documentation error.

**Comment by SenseException** [comment]
Looks like the name wasn't removed from Migrations while the DoctrineMigrationsBundle had this config key removed. Nice catch. Do you like to contribute a pull request for this?

**Comment by shepard8** [comment]
Sure, I left it just above. Thank you for inviting me to contribute!

**Comment by goetas** [comment]
This has been fixed in https://github.com/doctrine/migrations/commit/b9606574b15849ede8f8aa1c7f8357e9bd8a1609

It will be merged to master soon

**Comment by shepard8** [comment]
> This has been fixed in [b960657](https://github.com/doctrine/migrations/commit/b9606574b15849ede8f8aa1c7f8357e9bd8a1609)

I don't agree, the commit you mention fixes the `UPGRADE.md` file. The problem this issue is about is in the main documentation, which probably deserves to be updated too.
